RFCs/0007-Editors-API-Specification.md

```diff
@@ -111,6 +111,26 @@ At this point the initial handshake between Editor and Test Platform is
 complete. Above operation is required only once per launch of test platform.
 Subsequent discovery, execution operation don't require an initialization.
 
+### Initialize Extensions (Request)
```

Comment:
Please add a note that this is required for `net4x` targets only.

Should we suggest the caller to pass output path of test project here? For most of the frameworks, test adapter is copied to output directory.